lesson13/homework_13_json.py

```diff
@@ -0,0 +1,30 @@
+# Завдання 2:
+# Провалідуйте, чи усі файли у папці ideas_for_test/work_with_json є валідними json.
+# результат для невалідного файлу виведіть через логер на рівні еррор у файл json__<your_second_name>.log
+
+import logging
+import json
+from pathlib import Path
+
+logging.basicConfig(
+    filename='json__ponomarov.log',
+    level=logging.ERROR,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+
+folder_path = Path('https://github.com/dntpanix/automation_qa/tree/0c61017dfea127e5e4a048bfc411f07e1c324b27/ideas_for_test/work_with_json')
```

Comment:
What we will do if we have no internet or smth wrong with GitHub? How we will have an access to the file then?